pages/iot-hub/api-cli/getting-started-with-iot-hub-api.mdx

```diff
@@ -8,7 +8,7 @@ tags: iot iot-hub io-hub-api api curl mqtt jq json
 import Requirements from '@macros/iam/requirements.mdx'
```

Comment:
Github won't let me comment the lines  2 to 6: update mentions of API/curl to CLI